srcpkgs/libfido2/INSTALL.msg

```diff
@@ -0,0 +1,3 @@
+Group access to FIDO2 devices has been changed to group "plugdev".
+If you are not using elogind as your login manager, make sure to
```

Comment:
nitpick: elogind is a session manager, a login manager is something like lightdm/sddm/gdm